news/_posts/2023-10-10-release-20231010.md

```diff
@@ -0,0 +1,44 @@
+---
+kind: article
+title: "Release 20231010 and the Community"
+author: "The OpenRA Team"
+created_at: 2023-10-10 18:00 +0000
+disqus_id: "release-20231010"
+permalink: "/news/release-20231010/"
+---
+
+With two months of successful playtesting behind us, we are pleased to finally announce [Release 20231010]({{ '/download' | relative_url }}), which brings the public release up to date with development changes from November 2022 through to August 2023. The new release contains some long-awaited UI and UX improvements and many polish changes overall. It also serves as a jump-off point for many of the larger upcoming features, so do expect great things in the future!
+
+<figure>
+   <video autoplay loop muted>
+    <source src="{{ '/images/news/20231010-darkcolors.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/news/20231010-darkcolors.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20231010-darkcolors.png' | relative_url }}"/>
+  </video>
+  <figcaption>Restored the ability to select darker player colors.</figcaption>
```

Comment:
This needs to be
```suggestion
  <figcaption>The ability to select darker player colors has been restored.</figcaption>
```

to read as a functioning sentence/caption.

Reply:
sure